src/make-twig-include.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Generate a twig source string of an include with args, i.e.
+ * {% include '@cloudfour/components/button.twig' with { "type": "disabled" } only %}
+ * @param {string} path
+ * @param {any} args
+ */
+export const makeTwigInclude = (path, args) => {
+  const argsString =
+    Object.keys(args).length > 0
+      ? ` with ${JSON.stringify(args, null, 2)}`
+      : '';
+  return `{% include '${path}'${argsString} only %}`;
```

Comment:
Is there ever a use case where we wouldn't want to use `only`? 🤔 

If so, we might want to add a param to control this.

Reply:
I think it is best practice for includes to avoid implicitly passing through variables. For our docs, I put the `only` there so that if someone copy-pastes it it will probably "do the right thing", and they have to opt-in to the implicit passing through behavior. I can't think of any of our demos where someone using our patterns would always or usually want to have the implicit passing through behavior.

Reply:
Yeah that makes sense :+1:

Reply:
I think that makes sense. We can always revisit later, and this is probably a safer starting point than leaving it off.